### PR TITLE
visuallly still looks a bit ass but returns the list of users

### PR DESCRIPTION
**Pull request by AnthonyClarke25**

### CODE REVIEW

File: src/app/(dashboard)/chat/page.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"use client";
```
Comment (DMCFA) [inline src/app/(dashboard)/chat/page.tsx]: why do you need this on this file?

File: src/components/ChatUserList.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import getAllUsers from "@/data-access/users/get-all-users";
```
Comment (DMCFA) [inline src/components/ChatUserList.tsx:3]: add the use client here instead of the parent

File: src/components/ChatUserList.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import React, { useEffect, useState } from "react";
+import ChatUser from "./chatuser";
+import { BookUser } from "@/types/user";
+
+const ChatUserList = () => {
+  const [users, setUsers] = useState<BookUser[]>([]);
+
+  useEffect(() => {
```
Comment (DMCFA) [inline src/components/ChatUserList.tsx:11]: does it work if instead of the useEffect you just have
`const users = await getAllUsers()`

**Review by DMCFA — CHANGES_REQUESTED** [review]

File: src/components/ChatUserList.tsx (hunk 3 above)
Comment (AnthonyClarke25) [reply src/components/ChatUserList.tsx:11]: you can't use use effect with client components

**Review by DMCFA — APPROVED** [review]

**Pull request merged**